sdk/common/src/main/java/io/opentelemetry/sdk/common/InternalTelemetrySchemaVersion.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.common;
+
+/** Defines the self-monitoring telemetry SDK components should capture. */
+public enum InternalTelemetrySchemaVersion {
+  /** No self-monitoring metrics will be captured. */
+  DISABLED,
```

Comment:
It just occurred to me that we have two ways to disable internal telemetry:

- Set `InternalTelemetrySchemaVersion.DISABLED`
- Set the `Supplier<MeterProvider>` to a `MeterProvider::noop`

Probably should only have one way of doing this, which I think means getting rid of the DISABLED enum option. Sorry it looks like I gave you bad advice [here](https://github.com/open-telemetry/opentelemetry-java/pull/7265#discussion_r2069567357).